pages/data-providers/[data-provider-id]/deposit-dates.jsx

```diff
@@ -91,6 +90,41 @@ const DepositDates = ({
         )}
     </Card>
 
+    <Card tag="section">
+      <h2>Publication dates</h2>
+      <p>Compares publications dates with Crossref</p>
+      {store.depositDates.publicationDatesValidate ? (
+        <StackedVerticalBarChart
+          data={[
+            {
+              value: store.depositDates.publicationDatesValidate.fullCount,
+              caption: 'Matching',
+              background: 'var(--success)',
+            },
+            {
+              value: store.depositDates.publicationDatesValidate.partialCount,
+              caption: 'Incorrect',
```

Comment:
The data here is actually **correct** but it's not complete, i.e only year matches. _Incomplete_ sounds slightly better. Maybe the best way to call it is _Partially matches_.